OSDOCS-5367:adds console bugs to RNs for 4.13 GA (#58200)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[discrete]
[id="ocp-4-13-management-console-bug-fixes"]
==== Management Console
* Previously, user permissions were not checked when rendering the *Create Pod* button, and the button rendered for users without needed permissions. With this update, user permissions are checked when rendering the *Create Pod* button, and it renders for users for users with the needed permissions. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2005232[*BZ#2005232*])

* Previously, the `Pod` resource had a `PDB` *_add_*, *_edit_*, and *_remove_* actions in the Pod resource action menu that are not required. With this update, the actions are removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2110565[*BZ#2110565*])

* Previously, the `PodDisruptiojnBudget` field on the *Details* page had an incorrect help message. With this update, the help message is now more descriptive. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084452[*BZ#2084452*])

> * Previously, when navigating to the root path of the console, the URL redirects to the *Overview* page even if metrics are disabled and it doesn't appear in the navigation menu. With this update, when clicking the masthead logo or navigating to the root path of the console, the URL will redirect to the *project list* page if metrics are disabled. (link:https://issues.redhat.com/browse/OCPBUGS-3033[*OCPBUGS-3033*])

Reviewer: ```suggestion
* Previously, when navigating to the root path of the console, the URL redirected to the *Overview* page even if metrics were disabled and it did not appear in the navigation menu. With this update, when clicking the masthead logo or navigating to the root path of the console, the URL redirects to the *project list* page if metrics are disabled. (link:https://issues.redhat.com/browse/OCPBUGS-3033[*OCPBUGS-3033*])
```

> * Previously, the cluster dropdown was positioned so it is not always visible making it unclear which cluster you are viewing. With this update, the cluster dropdown is now in the masthead so the cluster dropdown is always visible, and you can always see which cluster you are viewing. (link:https://issues.redhat.com/browse/OCPBUGS-7089[*OCPBUGS-7089*])

Reviewer: ```suggestion
* Previously, the cluster dropdown was positioned so that it was not always visible, making it unclear which cluster you were viewing. With this update, the cluster dropdown is now in the masthead so the cluster dropdown is always visible, and you can always see which cluster you are viewing. (link:https://issues.redhat.com/browse/OCPBUGS-7089[*OCPBUGS-7089*])
```

> * Previously, the node progress bars were set to display when the cluster version had a status of `Failing`, `UpdatingAndFailing`, and `Updating` causing the node progress bars to display when the cluster is not updating. With this update, the node progress bars only display when the cluster version has a status of `UpdatingAndFailing` or `Updating`. (link:https://issues.redhat.com/browse/OCPBUGS-6049[*OCPBUGS-6049*])

Reviewer: ```suggestion
* Previously, the node progress bars were set to display when the cluster version had a status of `Failing`, `UpdatingAndFailing`, and `Updating`, causing the node progress bars to display when the cluster is not updating. With this update, the node progress bars only display when the cluster version has a status of `UpdatingAndFailing` or `Updating`. (link:https://issues.redhat.com/browse/OCPBUGS-6049[*OCPBUGS-6049*])
```

> * Previously, when downloading a `kubeconfig` file for a ServiceAccount, an error is displayed and the ServiceAccount token is unable to be reached. This error is due to the removal of autogenerated secrets. With this update, the download `kubeconfig` action has been removed and the error no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-7308[*BZ#OCPBUGS-7308*])

Reviewer: Not sure about "ServiceAccount" - probably "service account" or "`ServiceAccount`"

```suggestion
* Previously, when downloading a `kubeconfig` file for a ServiceAccount, an error was displayed and the ServiceAccount token was unable to be reached. This error was due to the removal of automatically generated secrets. With this update, the download `kubeconfig` action has been removed and the error no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-7308[*BZ#OCPBUGS-7308*])
```
